### ISSUE TITLE
show notes

### ISSUE THREAD

**Issue by hhrutter** [thechangelog/changelog.com#211]
I am already working on the transcripts for GoTimeFM.
I realized the show notes are not always accurate.
Is this smth that's open sourced as well because I can't find it.

**Comment by hhrutter** [comment]
Well,ok.. I just stumbled across a few episodes where the show notes
included entries from the previous one.
Not a big deal I guess and I am busy with the remaining past transcripts
per se anyways.

..but let's say if we had the show notes on github we'd had a way for
collaborators to make corrections over there.

**Comment by joebew42** [comment]
> Perhaps something you're interested in, @joebew42? ￼

Yeah! I could be interested in that, but I think we need more detail about that. To be honest, I have no a clear of what we are talking about. Open source a tool for handling the transcripts of the episodes? Or to allow a better transcripts editing, from within the admin ui?

**Comment by jerodsanto** [comment]
I could write up an actual spec. If you're not familiar with how our transripts work, check out our [transcripts repo](https://github.com/thechangelog/transcripts). You will find Markdown files for each show. Each time a file is added/edited on GitHub, the website refreshes its local copy of the transcript for that episode.

I imagine this system would work identically (on a different "show-notes" repo), but also have a feature that pushes back in the other direction. That way if we edit the show notes (on a published episode) via the admin, it would make that a commit on the GitHub repo.

**Comment by hhrutter** [comment]
maybe there is a way to have a markdown file for the show notes of each
episode organized like in the transcripts repo or maybe you can even stick
them in there
and have 2 files for each episode: the transcript itself and the
corresponding shownotes.
